Tests/test_pile_ui.gd

```diff
@@ -0,0 +1,43 @@
+extends GutTest
+#test for ui pop up
+
+var _card_container_scene: PackedScene = load("res://Cards/CardContainer.tscn")
+var _card_container_scroll_scene: PackedScene  = load("res://#Scenes/CardScrollUI.tscn")
+var _card_container: CardContainer = null
+var _card_scroll: Control = null
+
+const SCROLL_CONTAINER = 2
+const GRID_CONTAINER = 0
```

Comment:
not sure to understand what those two are as i didn't see them in other places

Reply:
Those are the indexes to get their node. I can the INDEX as a suffix.